akka-http-core/src/main/resources/reference.conf

```diff
@@ -107,6 +107,34 @@ akka.http {
     parsing = ${akka.http.parsing}
   }
 
+  host-connection-pool {
+    # The maximum number of parallel connections that a connection pool to a
+    # single host endpoint is allowed to establish. Must be greater than zero.
+    max-connections = 4
```

Comment:
This value is what you call "max-connections-per-host". There is no global limit.